commons/commons-model-extended-serialization/src/test/kotlin/uk/ac/ebi/extended/serialization/serializers/ExtFileSerializerTest.kt

```diff
@@ -22,12 +22,12 @@ class ExtFileSerializerTest(private val tempFolder: TemporaryFolder) {
         val file = tempFolder.createFile("test-file.txt", "content")
         val extFile = ExtFile(
             file = file,
-            fileName = "test/path/test-file.txt",
```

Comment:
I included this to cover the scenario of a file with an inner folder, I think it's worth having it.